canonical-web-and-design/cookie-policy
# Allow dynamic content for the cookie policy

## Done
The setup function accepts a string of content to replace the default content.

## QA
- Pull down this branch
- Run `yarn install` then `gulp build`
- Open index.html in the browser
- Check the cookie policy looks as it does on www.ubuntu.com
- Open index.html in an editor
- Pass a string to the setup function in the footer 
- Refresh the page and see that the content in the cookie policy has changed.
